charts/t8s-cluster/templates/workload-cluster/cloud-controller-manager.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: {{ printf "%s-ccm" .Release.Name }}
+  namespace: {{ .Release.Namespace}}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+spec:
+  chart:
+    spec:
+      chart: openstack-cloud-controller-manager
+      sourceRef:
+        kind: HelmRepository
+        name: {{ printf "%s-cloud-provider-openstack" .Release.Name }}
+        namespace: {{ .Release.Namespace }}
+      version: {{ printf "2.%d.x" (.Values.version.minor | int) }}
+  interval: 1h
+  kubeConfig:
+    secretRef:
+      name: {{ .Release.Name }}-kubeconfig
+  install:
+    remediation:
+      retries: -1
```

Comment:
I know we've used this a couple of times, but i never checked if the load is maybe rising through this.  
Does this have any impact on cpu usage?

Reply:
_Maayybee_, but even if, only during the initial install.

And as we don't have a choice here 🤷